example/lib/main.dart

```diff
@@ -21,18 +21,30 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'FlutterBleLib example',
       theme: new ThemeData(
-        primaryColor: new Color(0xFF0A3D91),
-        accentColor: new Color(0xFFCC0000),
+        primaryColor: Color(0xFF0A3D91),
+        accentColor: Color(0xFFCC0000),
+        scaffoldBackgroundColor: Color.fromRGBO(242, 242, 247, 1.0),
+        cardTheme: CardTheme(
```

Comment:
is it the same theme as for cards on peripherals list?

Reply:
Peripheral list does not utilize Card widget in the current implementation, but if it did - it should use the same style as the one on the details screen, hence definition here in the ThemeData.